app/_hub/kong-inc/request-transformer-advanced/index.md

```diff
@@ -136,7 +136,7 @@ params:
 
 ### Template as Value
 
-User can use any of the the current request headers, query params, and captured URI named groups as template to populate above supported config fields.
+Users can use any of the the current request headers, query params, and captured URI named groups as template to populate above supported config fields.
```

Comment:
Please make this improvement to prior versions of this documentation, eg. https://github.com/Kong/docs.konghq.com/blob/master/app/_hub/kong-inc/request-transformer-advanced/0.32-x.md